docroot/themes/humsci/humsci_basic/templates/components/global-footer.html.twig

```diff
@@ -0,0 +1,51 @@
+{#
+  /**
+  * @file
+  * Global Footer Component.
+  *
+  * Stanford Universal Footer as defined at https://identity.stanford.edu/web-mobile.html#build-something-new
+  * This component should be added into the footer region of your site below site-specific footer section, if any.
+  * The entire footer can be made to "stick" to the bottom of the browser window by including the @sticky-footer mixin.
+  */
+#}
+
+{% if template_path_logo is empty -%}
+  {%- set template_path_logo = "/themes/humsci/humsci_basic/templates/components/logo.html.twig" -%}
+{%- endif %}
+
+<section class="hb-global-footer su-global-footer {{ modifier_class }}">
+  <div class="su-global-footer__container">
+    <div class="su-global-footer__brand">
+      {%- include humsci_basic ~ template_path_logo -%}
+    </div>
+    <div class="su-global-footer__content">
+      <nav aria-label="global footer menu">
+        <ul class="su-global-footer__menu su-global-footer__menu--global">
+          {# The class of hb-global-footer__link has been added to all links to prevent a:not([class]) styles in humsci_basic/src/scss/elements/_base.scss from overriding Decanter styles #}
+          <li><a class="hb-global-footer__link" href="https://www.stanford.edu">Stanford Home</a></li>
+          <li><a class="hb-global-footer__link" href="https://visit.stanford.edu/plan/">Maps &amp; Directions</a></li>
+          <li><a class="hb-global-footer__link" href="https://www.stanford.edu/search/">Search Stanford</a></li>
+          <li><a class="hb-global-footer__link" href="https://emergency.stanford.edu">Emergency Info</a></li>
+        </ul>
+        <ul class="su-global-footer__menu su-global-footer__menu--policy">
+          <li><a class="hb-global-footer__link" href="https://www.stanford.edu/site/terms/"
```

Comment:
having a title for each of these seems very odd. The link text itself is pretty explanatory.

Reply:
also haha voiceover is not reading these titles

Reply:
oh interesting. when you hover you see these titles.

Reply:
That is interesting. This feels like the *legal* section. Perhaps the titles are a means to communicate and comply with certain legal standards.